ui/src/literature/components/ArxivPdfDownloadAction.jsx

```diff
@@ -1,24 +1,23 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 
+import { Button } from 'antd';
 import ListItemAction from '../../common/components/ListItemAction';
+import IconText from '../../common/components/IconText';
 import EventTracker from '../../common/components/EventTracker';
 
 class ArxivPdfDownloadAction extends Component {
   render() {
     const { arxivId } = this.props;
     const href = `//arxiv.org/pdf/${arxivId}`;
     return (
-      <EventTracker eventId="PdfDownload">
-        <ListItemAction
-          iconType="download"
-          text="pdf"
-          link={{
-            href,
-            target: '_blank',
-          }}
-        />
-      </EventTracker>
+      <ListItemAction>
+        <EventTracker eventId="PdfDownload">
+          <Button href={href} target="_blank">
```

Comment:
`ExternalLink` here.

Reply:
The ListItemAction was generating a button before, not an ExternalLink, I didn't want to change the behavior during the refactor